kolibri_explore_plugin/assets/src/components/AboutFooter.vue

```diff
@@ -2,25 +2,61 @@
 
   <b-container class="mb-5">
 
-    <b-row class="d-flex justify-content-center" cols="3">
-
-      <b-col class="col-auto d-flex justify-content-center">
-        <button
-          v-b-modal.about-modal
-          class="btn d-md-block d-none shadow-none"
-        >
-          {{ $tr('aboutLabel') }}
-        </button>
+    <b-row class="d-flex justify-content-between">
+
+      <b-col class="col-auto d-flex justify-content-start">
+        <b-container class="mb-5" fill-height>
+          <b-row class="d-flex justify-content-between" style="height: 100%">
+
+            <b-col class="col-auto d-flex justify-content-start">
+              <button
+                v-b-modal.about-modal
+                class="btn d-md-block d-none shadow-none"
+              >
+                {{ $tr('aboutLabel') }}
+              </button>
+            </b-col>
+
+            <b-col class="col-auto d-flex justify-content-end">
+              <button
+                v-b-modal.about-modal
+                class="btn d-md-block d-none shadow-none"
+                @click="$root.$emit('setAboutSection', 'privacy-policy-link')"
+              >
+                {{ $tr('privacyPolicyLabel') }}
+              </button>
+            </b-col>
+
+          </b-row>
+        </b-container>
       </b-col>
 
-      <b-col class="col-auto d-flex justify-content-center">
-        <button
-          v-b-modal.about-modal
-          class="btn d-md-block d-none shadow-none"
-          @click="$root.$emit('setAboutSection', 'privacy-policy-link')"
-        >
-          {{ $tr('privacyPolicyLabel') }}
-        </button>
+      <b-col class="col-auto d-flex justify-content-end">
```

Comment:
According to the requirements https://github.com/endlessm/endless-key-content-private/issues/64 this is only needed for the online instance. So this `<b-col>` should be added conditionally with a `v-if` https://vuejs.org/guide/essentials/conditional.html#v-if .

Currently there is no "is online instance" option in kolibri-explore-plugin. The only exception we do for online instance is the HIDE_DISCOVERY_TAB option. So either:

* add a new IS_ONLINE_INSTANCE option and derive "hide discovery tab" + "display Play/Microsoft buttons" from it, or
* add a new DISPLAY_STORE_BUTTONS option, to be set in the online instance only.

Reply:
Oops, I forgot about that requirement while diving into the vue rabbit hole

I’ve pushed a fixup which implements a third option: it uses the [app IDs configured on the PWA plugin](https://github.com/learningequality/kolibri/pull/10883) to determine whether to show the badges.

The rest of the changes in the fixup are a load of vue/bootstrap hackery to get things centred properly regardless of whether the badges are shown. There’s probably a better way of doing this.

Badges:
![buttons](https://github.com/endlessm/kolibri-explore-plugin/assets/53172/ebaa4c7c-8d78-4176-b94e-9d6f4e557117)


No badges:
![no buttons](https://github.com/endlessm/kolibri-explore-plugin/assets/53172/a508a435-101a-44d9-9fa0-c06ddb27c1eb)